mina.netty/src/main/java/org/kaazing/mina/core/service/AbstractIoAcceptor.java

```diff
@@ -106,7 +105,8 @@ public SocketAddress getLocalAddress() {
     @Override
     public final Set<SocketAddress> getLocalAddresses() {
         Set<SocketAddress> localAddresses = new HashSet<>();
-        synchronized (bindLock) {
```

Comment:
This file is pretty much same as in mina version. It uses both bondLock and boundAddresses locks